src/app/static/src/js/components/documents/file.vue

```diff
@@ -2,7 +2,9 @@
     <div>
         <file-icon></file-icon>
         <span>{{doc.display_name}}:</span>
-        <span v-if="doc.can_open"><a :href="doc.url + '?inline=true'">open</a>/</span><a :href="doc.url">download</a>
+        <a v-if="doc.can_open" :href="openUrl">open</a>
```

Comment:
I'm seeing a 'download' link for urls rather than 'open'. I think that's because 'can_open' is false for .url files as they're not in the list in canRenderInBrowser, and because .external is always falsy because it's not in the DocumentViewModel.